transport/src/main/java/io/netty/channel/SingleThreadIoEventLoop.java

```diff
@@ -34,10 +35,10 @@
 public class SingleThreadIoEventLoop extends SingleThreadEventLoop implements IoEventLoop {
 
     // TODO: Is this a sensible default ?
-    protected static final int DEFAULT_MAX_TASKS_PER_RUN = Math.max(1,
-            SystemPropertyUtil.getInt("io.netty.eventLoop.maxTaskPerRun", 1024 * 4));
+    private static final long DEFAULT_MAX_TASK_PROCESSING_QUANTUM_NS = TimeUnit.MILLISECONDS.toNanos(Math.max(100,
```

Comment:
Yeah, in CFS this should be the target latency within a process should get its guaranteed fair schedule/chance to run IIRC.
And here, have a similarish meaning i.e. I/O will get its chance to run after this, although with a delay (our tasks are not preemptive).

TBH what CFS does is to update via a timer (the OS tick rate) the virtual time of the task running on CPU, guaranteed a min granularity time to run on the CPU or eventually perform a different decision if there are more.